angular/src/app/workspace/simple-search/simple-search.component.html

```diff
@@ -15,7 +15,7 @@
   </div>
   <aside class="component-frame-aside">
     <ng-container *ngIf="(filterForm && dataTableModel)">
-      <form [formGroup]="filterForm" (ngSubmit)="onFilterFormSubmit()">
```

Comment:
**Change request:** Please keep this. This for the submit event.

Reply:
Because of the structure of the HTML, the button is no longer inside the form, so this doesn't do anything. I changed it to `(click)=onFilterFormSubmit()`, is that ok?

Reply:
A user can still press an enter/return key to trigger a submit event. That's why we still keep it.